### PR TITLE
CRM: Update Automations category labels: translate and capitalize automation

### PR DESCRIPTION
**Pull request by cleacos**
Fixes https://github.com/Automattic/zero-bs-crm/issues/3280

## Proposed changes:

This PR just updates the category names in the steps, translating and capitalizing them with the current implementation (string acting as a label). We are not going to use categories in this phase of the project, and we will probably create the Category class, which allows us to have more flexibility, like adding a proper slug, label, color, and icon.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
N/A

## Testing instructions:
- Check the code and the category names. They should be translated and capitalized.
- Run the test and check that they pass.

### CODE REVIEW

File: projects/plugins/crm/tests/php/automation/mocks/mock-class-contact-created-trigger.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ public static function get_description(): ?string {
 	 * @return string
 	 */
 	public static function get_category(): ?string {
-		return __( 'contact', 'zero-bs-crm' );
+		return 'contact';
```
Comment (gogdzl) [inline projects/plugins/crm/tests/php/automation/mocks/mock-class-contact-created-trigger.php:36]:
```suggestion
		return __( 'Contact', 'zero-bs-crm' );
```

File: projects/plugins/crm/tests/php/automation/mocks/mock-class-trigger-empty-slug.php [hunk 2]
```diff
@@ -33,7 +33,7 @@ public static function get_description(): ?string {
 	 * @return string
 	 */
 	public static function get_category(): ?string {
-		return __( 'contact', 'zero-bs-crm' );
+		return 'contact';
```
Comment (gogdzl) [inline projects/plugins/crm/tests/php/automation/mocks/mock-class-trigger-empty-slug.php:36]:
```suggestion
		return __( 'Contact', 'zero-bs-crm' );
```

**Review by gogdzl — CHANGES_REQUESTED** [review]
Looks like we missed two strings, could you check if we need to do the same for those?

File: projects/plugins/crm/tests/php/automation/mocks/mock-class-contact-created-trigger.php (hunk 1 above)
Comment (cleacos) [reply projects/plugins/crm/tests/php/automation/mocks/mock-class-contact-created-trigger.php:36]: It's tests context. We don't need to translate it there.

File: projects/plugins/crm/tests/php/automation/mocks/mock-class-trigger-empty-slug.php (hunk 2 above)
Comment (cleacos) [reply projects/plugins/crm/tests/php/automation/mocks/mock-class-trigger-empty-slug.php:36]: Same as before. Test context, we don't need to translate it.

**Review by gogdzl — APPROVED** [review]
LGTM!

**Comment by cleacos** [comment]
Tests are failing for PHP 8.1 and 8.2... Locally I'm getting a different error. Here are only warnings... I'm going to re-execute them...

**Comment by gogdzl** [comment]
Here's the log:

```
  In Assert.php line 2863:
                                                                                 
    Hey! Something seems wrong.                                              
    There are some PHP errors or notices:                                
                                                                                 
    Array                                                                        
    (                                                                            
        [0] => [Wed Aug 30 16:58:08 2023] PHP Warning:  Attempt to read propert  
    y "paid" on null in /tmp/wordpress-trunk/src/wp-content/plugins/crm/include  
        [38] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [39] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [40] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [41] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [42] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [43] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [44] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [45] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [46] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [47] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [48] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [49] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [50] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [51] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [52] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [53] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [54] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
        [55] => [Wed Aug 30 16:58:09 2023] PHP Warning:  Undefined property: st  
    dClass::$slug in /tmp/wordpress-trunk/src/wp-content/plugins/crm/includes/Z  
    eroBSCRM.AdminPages.php on line 2234                                         
                                                                                 
    )       
```

**Comment by cleacos** [comment]
It seemed something temporal... I merged trunk, and everything went well.

**Pull request merged**